x-pack/plugin/sql/qa/server/src/main/resources/setup_no_cols.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS "no_cols";
+CREATE TABLE "no_cols" ();
+
+INSERT INTO "no_cols" DEFAULT VALUES;
```

Comment:
Is this required?

Reply:
I also want to cover cases with empty documents indexed in the empty mapping index.

Reply:
Yes that's true. Do you think it would be worth to also have an `empty_empty_mapping` index for covering the no rows cases? To some extent, the "no rows" cases are already covered by tests that filter out all records.

Reply:
I don't know if that's too much, @costin, what do you think?